src/main/java/uk/gov/hmcts/divorce/caseworker/event/CaseworkerHwfEvidenceRequested.java

```diff
@@ -0,0 +1,45 @@
+package uk.gov.hmcts.divorce.caseworker.event;
+
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.stereotype.Component;
+import uk.gov.hmcts.ccd.sdk.api.CCDConfig;
+import uk.gov.hmcts.ccd.sdk.api.ConfigBuilder;
+import uk.gov.hmcts.divorce.divorcecase.model.CaseData;
+import uk.gov.hmcts.divorce.divorcecase.model.State;
+import uk.gov.hmcts.divorce.divorcecase.model.UserRole;
+
+import static uk.gov.hmcts.divorce.caseworker.event.page.UpdateContactDetails.TITLE;
+import static uk.gov.hmcts.divorce.divorcecase.model.State.AwaitingHWFDecision;
+import static uk.gov.hmcts.divorce.divorcecase.model.State.AwaitingHWFEvidence;
+import static uk.gov.hmcts.divorce.divorcecase.model.UserRole.CASE_WORKER;
+import static uk.gov.hmcts.divorce.divorcecase.model.UserRole.LEGAL_ADVISOR;
+import static uk.gov.hmcts.divorce.divorcecase.model.UserRole.SUPER_USER;
+import static uk.gov.hmcts.divorce.divorcecase.model.access.Permissions.CREATE_READ_UPDATE;
+
+@Component
+@Slf4j
+public class CaseworkerHwfEvidenceRequested implements CCDConfig<CaseData, State, UserRole> {
+
+    public static final String CASEWORKER_HWF_EVIDENCE_REQUESTED = "caseworker-hwf-evidence-requested";
+
+
+
+    @Override
+    public void configure(final ConfigBuilder<CaseData, State, UserRole> configBuilder) {
+        configBuilder
+            .event(CASEWORKER_HWF_EVIDENCE_REQUESTED)
+            .forStateTransition(AwaitingHWFDecision, AwaitingHWFEvidence)
+            .name("HWF evidence required")
+            .description(TITLE)
```

Comment:
is this correct ?

Reply:
Hi @nitinprabhu thanks for the comment the .name should be "HWF evidence required" as mentioned on the ticket, the description however should not be TITLE, it should just be "HWF evidence required" I believe, I'll update that.

Reply:
@nitinprabhu I updated the PR.